.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/k8s-frameworkcontroller/src"]
+	path = src/k8s-frameworkcontroller/src
+	url = https://github.com/microsoft/frameworkcontroller
```

Comment:
why need submodule

Reply:
current framework controller is built in other place, need the source if we want to build the image with other pai services during deployment

Reply:
Do we plan to build it along with PAI?

Reply:
I think we need to build it like yarn launcher in the future, not the first stage in alpha release.
Adding it as a git submodule is the best way to fix version as the code is in another repo.

Reply:
We (Bin and Fan) previously decided to decouple the DevOps lifecycle among FC, HS(Hived) and PAI. And PAI should directly use a specific released image version of FC and HS. Is that fine?